src/openeo_gfmap/fetching/commons.py

```diff
@@ -100,11 +137,11 @@ def _load_collection(
             spatial_extent, BoundingBoxExtent
         ), "Please provide only a bounding box for tile based fetching."
         spatial_extent = dict(spatial_extent)
-        cube = connection.load_collection(
-            collection_id=collection_name,
+        cube = load_collection_method(
```

Comment:
the cube definition seems to be the same for each fetchtype. Could we extract these from the if/elifs to make it more readable?

Reply:
Also to be addressed in #132